packages/es-components/src/components/controls/buttons/IconButton.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Icon from '../../base/icons/Icon';
+
+const ContainerButton = styled.button`
+  align-items: center;
+  background-color: transparent;
+  border: none;
+  cursor: ${props => (props.disabled ? '' : 'pointer')};
+  display: flex;
+  flex-direction: column;
+  font-family: inherit;
+
+  i {
+    background-color: ${props =>
+      props.isHighlighted ? props.theme.brandColors.primary3 : 'white'};
+    color: ${props =>
+      props.isHighlighted ? 'white' : props.theme.brandColors.primary3};
+  }
+  &:active:enabled i {
+    background-color: ${props =>
+      !props.isHighlighted ? props.theme.brandColors.primary3 : 'white'};
+    color: ${props =>
+      !props.isHighlighted ? 'white' : props.theme.brandColors.primary3};
+  }
+
+  span {
+    font-weight: ${props => (props.isHighlighted ? 'bold' : 'normal')};
+  }
+  &:active:enabled span {
+    font-weight: ${props => (!props.isHighlighted ? 'bold' : 'normal')};
+  }
+`;
```

Comment:
Do we have any kind of hover state for this?

Reply:
Not at this time. That might be something we want eventually, but the initial use case doesn't call for it.

Reply:
Maybe not this iteration, but I think we need to push back with design and add a hover state to this. In the current state I can't tell if this is even a button at all.